### PR TITLE
Homework#5

### PR DESCRIPTION
**Pull request by MarkStarikov**

### CODE REVIEW

File: Homework5_1/src/Homework5_1_1/Main.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class Main {
+    public static void main(String[] args) {
+        String randomCity = "Brno";
+        List<String> cities = new ArrayList<>(Arrays.asList("Olomouc", "Lodz,", "Brno", "Ploesti", "Varna", "Brno", "Trnava", "Kluzh", "Lodz", "Rzheshuv", "Brashov", "Brno", "Mishkolts", "Brailov", "Kishvarda", "Chelm", "Kluzh", "klodzko", "Belostok", "Ostrava", "Gdunya"));
+        countOccurrence(cities, randomCity);
+        System.out.println("Quantity of repeat city Brno are " + countOccurrence(cities, randomCity));
+    }
+
+    static int countOccurrence(List<String> cities, String randomCity) {
+        int b = 0;
```
Comment (shaitanaleksander) [inline Homework5_1/src/Homework5_1_1/Main.java:15]: bad var name

File: Homework5_1/src/Homework5_1_2/Main.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+     String [] chemicalElements = new String[] {"Sodium", "Iron", "Oxygen", "Lead", "Neon", "Tin", "Nickel", "Zinc", "Gold", "Silver", "Copper", "Uranus", "Carbon", "Nitrogen"};
+        for (int i = 0; i < chemicalElements.length; i++) {
+            System.out.println(chemicalElements[i]);
+        }
+         List<String> elements = toList(chemicalElements);
+        System.out.println(elements);
+    }
+    public static <String> List<String> toList(String[] chemical){
+        List<String> elements = new ArrayList<>();
+        for (String i: chemical){
```
Comment (shaitanaleksander) [inline Homework5_1/src/Homework5_1_2/Main.java:16]: i is a bad name

File: Homework5_1/src/Homework5_1_3/Main.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List <Integer> numbers = new ArrayList<>(Arrays.asList( 5, 8483, 59, 32, 19, 75, 59, 43, 19, 3, 43, 3, 59, 8, 99, 8483));
+         System.out.println(findUnique(numbers));
+    }
+        public static List <Integer> findUnique(List<Integer> numbers){
+            List<Integer> clone = new ArrayList<>();
+            for (Integer i: numbers){
```
Comment (shaitanaleksander) [inline Homework5_1/src/Homework5_1_3/Main.java:15]: i and clone is a bad names  for var

File: Homework5_1/src/Homework5_1_4/Main.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List<String> vitamins = new ArrayList<>(Arrays.asList("A", "C", "E", "D", "B12", "K", "B5", "D", "B9", "C", "P", "E",  "B5", "D", "B9", "C", "P", "A", "B12", "P", "B9", "B5", "P", "E"));
+        calcOccurance(vitamins);
+    }
+    static public void calcOccurance(List<String> vitamins) {
+        List<String> cloneUnique = new ArrayList<>();
+        for (String i: vitamins){
```
[the remark this answers is not in the document]
Comment (MarkStarikov) [reply Homework5_1/src/Homework5_1_4/Main.java:15]: I thought, in cycles "for" always must be use "i". Becose it reduction form of "index" . You told about it yeasterday too

**Pull request merged**